src/scss/2022/components/_section-header.scss

```diff
@@ -0,0 +1,56 @@
+@use "../tools/typography";
+@use "../tools/svg" as *;
+@use "../tools/color" as *;
+@use "../tools/space" as *;
+
+.section-header {
+  padding-top: space('10vh');
+  position: relative;
+  
+  &::after {
+    content: '';
+    display: block;
+    height: min(40vmin, 26rem);
+    width: min(40vmin, 26rem);
+    background: radial-gradient(circle at 200% 50%,
+                                hsla(var(--color-pink-1-val), 1) 0%,
+                                hsla(var(--color-pink-2-val), 1) 60%,
+                                hsla(var(--color-black-val), 0) 95%)
+                                no-repeat;
```

Comment:
This is so cool. Elegant solution.

Reply:
Another cheer for this - I appreciate how the blend modes interact as the diamond passes over the circle.